test/fetch/data-uri.js

```diff
@@ -121,7 +119,7 @@ test('https://mimesniff.spec.whatwg.org/#parse-a-mime-type', (t) => {
   t.same(parseMIMEType('text/html;charset="shift_jis"iso-2022-jp'), {
     type: 'text',
     subtype: 'html',
-    parameters: new Map([['charset', '"shift_jis"']])
+    parameters: new Map([['charset', 'shift_jis']])
```

Comment:
not really sure where I got the quotes from but this is even the wrong output as shown in https://mimesniff.spec.whatwg.org/#example-mime-type-parser-trailing-garbage